redbot/core/commands/context.py

```diff
@@ -237,3 +237,12 @@ def embed_color(self):
             )
         else:
             return await self.send(message)
+
+    @property
+    def clean_prefix(self) -> str:
+        """str: The command prefix, but a mention prefix is displayed nicer."""
+        if self.guild is not None:
+            me = self.guild.me
+        else:
+            me = self.bot.user
+        return self.prefix.replace(me.mention, f"@{me.display_name}")
```

Comment:
I think it might be worth adding `me` to `context` as
```py
@property
def me(self):
    return self.guild.me if self.guid else self.bot.me
```
And use that here inside clean_prefix. There are many cases where the user/member object may be desired, and such a shortcut is nice to have.